odata-csdl/14 Vocabulary and Annotation.md

```diff
@@ -1309,6 +1309,18 @@ Example ##ex:
 Dynamic expressions allow assigning a calculated value to an applied
 term.
 
+If a calculated value is not acceptable for the type of the term or
+its [facets](#TypeFacets), does not meet the constraints imposed by its
+: varjson
+`$Nullable` property
+:
+: varxml
+`Nullable` attribute
+:
+or by its
+annotations from the Validation vocabulary [OData-VocValidation](#ODataVocValidation),
+then the term is effectively not applied (see [example ##termdefault]).
```

Comment:
I think we want to say the client can make no assumptions, because the server tried to say something about the term but it was not valid.